src/main/java/com/android/volley/AsyncCache.java

```diff
@@ -0,0 +1,42 @@
+package com.android.volley;
+
+import androidx.annotation.Nullable;
+
+/**
+ * AsyncCache is an abstract class which implements the Cache interface for backwards compatibility,
```

Comment:
nit: update Javadoc now that we no longer implement Cache. My suggestion would just be:

/** Asynchronous equivalent to the {@link Cache} interface. */

As that is a simple summary of what this class is, and links to Cache in a more structured form so you can leverage the existing docs there if someone is looking for more detail.